api/proto/common/common.proto

```diff
@@ -2,6 +2,10 @@ syntax = "proto3";
 package common;
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common";
 
+// TODO: I generated the comments in this file using copilot. Reviewers, please verify accuracy,
+// and after you do I will remove this comment. This PR should not merge until this comment is removed.
+
+// A KZG commitment to a G1 point.
```

Comment:
I think G1 point *is* a commitment, not that this is a commitment *to* G1

Reply:
Changed to `// A KZG commitment`